long_term/schema.sql

```diff
@@ -1 +1,54 @@
--- SQL to initialise database.
\ No newline at end of file
+-- SQL to initialise database.
+
+DROP TABLE IF EXISTS reading;
+DROP TABLE IF EXISTS plant;
+DROP TABLE IF EXISTS origin_location;
+DROP TABLE IF EXISTS botanist;
+
+CREATE TABLE botanist(
+    botanist_id SMALLINT GENERATED ALWAYS AS IDENTITY,
+    first_name VARCHAR(255) NOT NULL,
+    last_name VARCHAR(255) NOT NULL,
+    email VARCHAR(255) NOT NULL UNIQUE,
+    phone VARCHAR(30) NOT NULL,
```

Comment:
Add unique constraint. I don't think two botanists should have same number+extension